src/App.js

```diff
@@ -1,28 +1,20 @@
-import React, { useState, useEffect } from 'react';
+import React from 'react';
 
 import './App.scss';
 
 import Header from './Components/Header';
 import Content from './Components/Content';
 import Footer from './Components/Footer';
-import Home from './Components/Pages/Home/';
-import OfferList from './Components/OfferList';
+import Home from './Components/Pages/Home';
+import List from './Components/Pages/List';
 
 function App() {
-  const [offers, setOffers] = useState([]);
-
-  useEffect(() => {
-    fetch(`api/categories/1/offers`)
-      .then(response => response.json())
-      .then(data => setOffers(data));
-  }, []);
-
   return (
     <div className="App">
       <Header />
       <Content>
         <Home path="/" />
-        <OfferList path="/offers" offers={offers} />
+        <List path="/categories/:categoryID" />
```

Comment:
Ten URL `/categories/:categoryId` nie pasuje tutaj, bo to jest lista ofert z danej kategorii. Tak samo nazwa strony `List` też jest zbyt ogólna w tym przypadku.